packetery/template/carrier/country.latte

```diff
@@ -1,6 +1,7 @@
 <div class="packetery-options-page">
 	<h1>{__( 'Packeta', 'packetery' )} - {__( 'Country options', 'packetery' )}: {$country_iso|upper}</h1>
 
+	{php do_action( 'packetery_admin_notices', 'carrier-country' )}
```

Comment:
Nešlo by místo php do_action v šabloně udělat do_filter při přípravě parametrů pro šablonu? Třeba bychom se tak mohli zbavit i následujícího řádku.

Reply:
Smyslem MessageManagera je zjednodušit práci se zprávami. V latte se musí určit místo, kde se vykreslí ty zprávy. Když se to bude dělat přes do_action, tak to bude konsistentní s WordPress admin_notices v admin-header.php. Třeba options form to dělá podobně, jen používá jiný systém. Snažit se o čistotu v tomto případě mně nedává smysl, když už v souboru voláme překladové a jiné funkce.